dirsrvtests/tests/suites/pwp_storage/storage_test.py

```diff
@@ -106,27 +389,27 @@ def test_check_two_scheme(topo):
     user.delete()
 
 @pytest.mark.skipif(ds_is_older('1.4'), reason="Not implemented")
-def test_check_pbkdf2_sha256(topo):
-    """Check password scheme PBKDF2_SHA256.
+def test_check_pbkdf2_sha512(topo):
```

Comment:
Should we really remove the test about the old scheme ?
IMHO it should be better to keep it and add a similar test for the new scheme

Reply:
Same here, agreed.

Reply:
Okay, first of all, the initial test case was invalid (it removed the `PBKDF2-SHA256` plugin, but it checked for `PBKDF2_SHA256`).

I tried to make the test about both `PBKDF2-SHA256` first and then about `PBKDF2_SHA256` consistently. And if we set the password scheme to it and restart it, the plugin entry will not be bootstraped.

It seems intentional, as only the latest recommended password plugin is bootstrapped correctly: https://github.com/389ds/389-ds-base/blob/main/ldap/servers/slapd/config.c#L42-L50

So, I changed the test to `PBKDF2-SHA512`.

P.S. the test in `dirsrvtests/tests/suites/password/pbkdf2_upgrade_plugin_test.py` checks for PBKDF2-SHA256, though and it is restored if it's not set to password storage scheme in cn=config